README.md

```diff
@@ -231,6 +231,7 @@ This will spin up 3 c3.large instances and start the com.FooTest simulation on e
 | ec2.key.pair.name | gatling-key-pair | The [key pair](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) that will be used to access the load generator instances. |  | 1.0.0 |
 | ec2.security.group | gatling-security-group | The [security group](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-network-security.html) used by the load generators. Make sure that your local development environment and CI system have SSH access to the load generators via the security group.  |  | 1.0.0 |
 | ec2.force.termination | false | `true` if the load generators should be terminated after the test despite of test failures. `false` if the load generators should be kept alive in case of failures to allow troubleshooting. If the tests succeeds all load generators will be terminated regardless of this setting. Setting it to `false` allows troubleshooting failed load generators. Setting it to `true` does not allow troubleshooting but helps to reduce EC2 costs. | | 1.0.4-SNAPSHOT |
+| ec2.keep.alive | false | `true` if the load generators should be left running for another test (will override ec2.force.termination. `false` to allow load generators to be terminated. | If tests are not going to run for the full hour use this setting to keep from being charged for the 1 hour minimum up time, also cuts down on load generator initializing time | |
```

Comment:
please mention that this is available since version 1.0.4-SNAPSHOT. i will change to 1.0.4 after releasing the new version.